services/vehicle-assembly-process-defect-detection-model-service/app/core/middleware.py

```
@@ -0,0 +1,489 @@
+- 요청 제한 (Rate Limiting)
+"""
+
+import time
+import uuid
+import logging
+from typing import Callable, Dict, Any
+from datetime import datetime, timedelta
+from collections import defaultdict
+
+from fastapi import FastAPI, Request, Response
+from fastapi.middleware.cors import CORSMiddleware
```

Ruff (0.12.2)

`fastapi.middleware.cors.CORSMiddleware` imported but unused

Remove unused import: `fastapi.middleware.cors.CORSMiddleware`

(F401)